33/ops.html

```diff
@@ -3401,6 +3401,121 @@ <h4 class="anchor-heading"><a id="zkops" class="anchor-link"></a><a href="#zkops
     <li>Don't overbuild the cluster: large clusters, especially in a write heavy usage pattern, means a lot of intracluster communication (quorums on the writes and subsequent cluster member updates), but don't underbuild it (and risk swamping the cluster). Having more servers adds to your read capacity.</li>
   </ul>
   Overall, we try to keep the ZooKeeper system as small as will handle the load (plus standard growth capacity planning) and as simple as possible. We try not to do anything fancy with the configuration or application layout as compared to the official release as well as keep it as self contained as possible. For these reasons, we tend to skip the OS packaged versions, since it has a tendency to try to put things in the OS standard hierarchy, which can be 'messy', for want of a better way to word it.
+
```

Comment:
There were some more changes in this file from the following PR https://github.com/apache/kafka/pull/12679
Have those been pushed to the site already?

Reply:
did those changes get backported to the 3.3 branch?

Reply:
I do see it on the 3.3 branch -- https://github.com/apache/kafka/commit/2e952805431f9d4a7deef5c7f8453c63aa9702db